apis/ops/v1alpha1/mongodb_ops_types.go

```diff
@@ -71,10 +71,14 @@ type MongoDBOpsRequestSpec struct {
 	Restart *RestartSpec `json:"restart,omitempty"`
 	// Specifies information necessary for reprovisioning database
 	Reprovision *Reprovision `json:"reprovision,omitempty"`
+
 	// Specifies the Readiness Criteria
 	ReadinessCriteria *MongoDBReplicaReadinessCriteria `json:"readinessCriteria,omitempty"`
 	// Timeout for each step of the ops request in second. If a step doesn't finish within the specified timeout, the ops request will result in failure.
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
+	// ApplyOptions is to control the execution of OpsRequest depending on the database state.
+	// Default is `ApplyIfReady`
+	ApplyOptions *ApplyOptions `json:"applyOptions,omitempty"`
```

Comment:
`Default is `ApplyIfReady``

This should not be a pointer. Just use default value.